blazor/breadcrumb/overflow.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Breadcrumb Overflow with Blazor Breadcrumb component | Syncfusion
-description: Overflow section in Blazor Breadcrumb explains how to limit the number of Breadcrumb items to be displayed.
+description: Checkout and learn here all about overflow in Blazor Breadcrumb component of Syncfusion, its element, and more.
```

Comment:
Change like this - Checkout and learn here all about overflow in Syncfusion Blazor Breadcrumb component, its element, and more.

Reply:
changed